tests/functional/common.bash

```diff
@@ -20,6 +20,20 @@ VERBS=(
 )
 FIXTURES="$BATS_TEST_DIRNAME/fixtures"
 
+common.is_stopped() {
+    local workdir="${1?}"
+    local status=$(
+      "$LAGOCLI" \
+          --out-format flat \
+          status
+    )
+    echo "$status" | grep -q "status: up"
+    if [[ $? -ne 0 ]]; then
+        return 0
+    else
+        return 1
+    fi
+}
```

Comment:
you need to add the verb in ``common.bash`` also in https://github.com/lago-project/lago/blob/master/tests/functional/common.bash#L3